src/fetch/payouts.py

```diff
@@ -219,13 +231,22 @@ def as_payouts(self) -> list[Transfer]:
             return result
 
         try:
-            result.append(
-                Transfer(
-                    token=None,
-                    recipient=self.solver,
-                    amount_wei=reimbursement_eth,
+            if self.bonding_pool == COW_BONDING_POOL:
+                result.append(
+                    Transfer(
+                        token=None,
+                        recipient=self.reward_target,
+                        amount_wei=reimbursement_eth,
+                    )
+                )
+            else:
+                result.append(
+                    Transfer(
+                        token=None,
+                        recipient=self.solver,
+                        amount_wei=reimbursement_eth,
+                    )
                 )
```

Comment:
This whole chunk would be a lot simpler if you just:

In pseudo python

Recipient = self.rewardtarget if (your condition) else self.solver

Reply:
Yeap, that's a fair point. Just rewrote it the correct way. Thanks!